config/cfme/manifest_5_11.json

```diff
@@ -24,58 +24,64 @@
       "vms": {
         "id": null,
         "name": null,
+        "description": null,
         "type": null,
-        "guid": null,
+        "ems_ref": null,
         "uid_ems": null,
         "archived": null,
-        "cpu_cores_per_socket": null,
+        "orphaned": null,
```

Comment:
I don't think that it makes sense to ask if a VM is orphaned when you're essentially doing `ems.vms` given that orphaned/archived means the ems_id is nil

Reply:
Fair enough, I'm used to Vm.all where you do need to check these, I hadn't realised that the ems_id was nil in that case. I'll remove